scripts/bash/pm2startInline.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+
+if [ -z "$1" ]; then
+  echo "Usage: yarn start-dev <stack>"
+  echo ""
+  echo "All stacks:"
+  ls -1 ../../packages/devops/configs/pm2/ | sed 's|.config.js||g' | grep -v 'base' | awk '$0="  "$0'
+  echo ""
+  echo "Tips:"
+  echo "  - normal pm2 commands work e.g. yarn pm2 status"
+  echo "  - start multiple stacks by calling this cmd multiple times"
+  exit 1
+fi
+
+yarn pm2 start "../../packages/devops/configs/pm2/$1.config.js"
+
+# When user quits logs stop everything
+trap "echo Stopping... && yarn pm2 delete all" EXIT
```

Comment:
One awkwardness with this one, if a user is running multiple apps at once (tupaia-web and admin-panel), and then stops admin-panel, everything gets stopped and they have to stop and restart tupaia-web.

Not sure if there's an easy fix for that though... and running multiple apps isn't too common.

Reply:
Yeah I had a look into this, there isn't really a nice way of doing this.

IMO given how its quick and easy to restart the one you want to keep up, and how rare it is to slim down what you're working on, its a minor inconvenience.

If you like I think we can do it by calling multiple stacks at the same time though, and generating a dynamic ecosystem file, e.g. `yarn start-stacks tupaia-web admin-panel`. Want me to do that?

Reply:
If we started multiple at once, killing one would still kill both though yeah? Since there's just one command that's being run.

I agree with your reasoning that it's a minor inconvenience, so happy to leave as is :)

Reply:
Ah yeah that's right, so basically same result, good point!